doc/configuration-v2_2-experimental.md

```diff
@@ -29,11 +29,13 @@ The Ignition configuration is a JSON document conforming to the following specif
       * **_start_** (integer): the start of the partition (in device logical sectors). If zero, the partition will be positioned at the earliest available part of the disk.
       * **_typeGuid_** (string): the GPT [partition type GUID][part-types]. If omitted, the default will be 0FC63DAF-8483-4772-8E79-3D69D8477DE4 (Linux filesystem data).
       * **_guid_** (string): the GPT unique partition GUID.
+      * **_wipePartition_** (boolean) whether or not to delete the partition before creation. TODO(ajeddeloh): fix docs to have filesystem, partition, and raid reuse semantics instead of just filesystem
   * **_raid_** (list of objects): the list of RAID arrays to be configured.
     * **name** (string): the name to use for the resulting md device.
     * **level** (string): the redundancy level of the array (e.g. linear, raid1, raid5, etc.).
     * **devices** (list of strings): the list of devices (referenced by their absolute path) in the array.
     * **_spares_** (integer): the number of spares (if applicable) in the array.
+    * **overwriteDevices** (boolean): whether or not to create a new raid overwriting an existing one. TODO(ajeddeloh): fix docs to have filesystem, partition, and raid reuse semantics instead of just filesystem
```

Comment:
`...to wipe the devices before RAID creation` would be consistent with the filesystem wording.

Drop TODO.

Why `overwrite` rather than `wipe`?

Reply:
I thought the FS code actually explicitly deleted the old FS first; it doesn't: it overwrites it. I would be OK with `wipeDevices` since we're already using `wipe` to mean overwrite.